code/controllers/subsystem/goldmansachs.dm

```diff
@@ -0,0 +1,31 @@
+SUBSYSTEM_DEF(goldmansachs)
+	name = "Economy"
+	flags = SS_NO_INIT
+	var/paycheck_interval = 1 MINUTES
+	var/roundstart_paychecks = 5
+	var/budget_pool = 25000
```

Comment:
Consider making this configurable perhaps? Maybe dependant on pop/map/server setting?

Reply:
This is your cargo budget and R&D points too.

Reply:
Even more reason to make it configurable so we don't have to recode the darn thing every balance change.

Or perhaps some maps get bigger budgets than others or some rounds the station is given a shoestring budget as a kind of mini event.

Many possibilities.

Reply:
Admins don't need control over balance.

Reply:
If it's in a subsystem we can edit it via the master control panel.

Reply:
^